### PR TITLE
Feature multiselect

### PR DESCRIPTION
**Pull request by m-nab**
Updated feature-request for #35

### CODE REVIEW

File: Example/Example.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
```
Comment (shibayan) [inline Example/Example.csproj]: Please revert this change

File: Sharprompt/Sharprompt.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+     <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (shibayan) [inline Sharprompt/Sharprompt.csproj]: Please fix indent

File: Example/Example.csproj (hunk 1 above)
Comment (m-nab) [reply Example/Example.csproj]: Thank you for your feedback. This should be fixed now.

File: Sharprompt/Sharprompt.csproj (hunk 2 above)
Comment (m-nab) [reply Sharprompt/Sharprompt.csproj]: Thank you for your feedback. This should be fixed now.

**Comment by shibayan** [comment]
Thank you for contributes.
